tests/ossfuzz/config/fuzz_lzip.dict

```diff
@@ -0,0 +1 @@
+"LZIP"
```

Comment:
This dictionary could also include valid lzip header bytes for versions 0 and 1. See the [file format](https://www.nongnu.org/lzip/lzip_talk_ghm_2019.html) for what these bytes should be.